engine/src/tilecachegl.cpp

```diff
@@ -555,6 +555,68 @@ bool MCTileCacheOpenGLCompositor_BeginLayer(void *p_context, const MCRectangle&
         self -> is_blending = true;
         glEnable(GL_BLEND);
     }
+    
+    switch(p_ink)
+    {
+        case GXblendClear:
+        case GXclear:
+            glBlendFunc(GL_ZERO, GL_ONE_MINUS_SRC_ALPHA);
+            break;
+        case GXblendSrc:
+            glBlendFunc(GL_ONE, GL_ZERO);
+            break;
+        case GXblendDst:
+            glBlendFunc(GL_ZERO, GL_ONE);
+            break;
+        case GXblendSrcOver:
```

Comment:
These should be moved to the default case.

Reply:
done